bom/pom.xml

```diff
@@ -29,7 +29,7 @@
   <description>A &quot;bill-of-materials&quot; for Hono.</description>
 
   <properties>
-    <artemis.image.name>enmasseproject/activemq-artemis:2.2.0-3</artemis.image.name>
+    <artemis.image.name>enmasseproject/activemq-artemis:2.2.0-4</artemis.image.name>
```

Comment:
my understanding is that this one does no longer include JGroups, right?

Reply:
Yes, and it has been already approved https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15347